application_sandbox/pipeline_creation_feedback/main.cpp

```diff
@@ -171,8 +171,9 @@ class CubeSample : public sample_application::Sample<CubeFrameData> {
           "VK_PIPELINE_CREATION_FEEDBACK_BASE_PIPELINE_ACCELERATION_BIT_EXT");
     }
     app()->instance()->GetLogger()->LogInfo("duration: ", pcf.duration);
-    app()->instance()->GetLogger()->LogInfo("PipelineStageCreationFeedback:");
     for (int i = 0; i < kNumStages; i++) {
+      app()->instance()->GetLogger()->LogInfo(
+          "PipelineStageCreationFeedback (Stage ", i, ")");
```

Comment:
nit: the colon was dropped.
"PipelineStageCreationFeedback (Stage ", i, "): ");

Reply:
Done.